docs/instrumentation-list.yaml

```diff
@@ -6535,6 +6656,12 @@ libraries:
           type: STRING
   jsf:
   - name: jsf-mojarra-1.2
+    display_name: JSF
+    description: |
+      This instrumentation enables controller spans for Mojarra JSF action listeners (controller spans are disabled by default).
+    library_link: https://github.com/eclipse-ee4j/mojarra
+    features:
+    - CONTROLLER_SPANS
```

Comment:
besides controller spans jsf instrumentations also update the server span name, but they don't update the route

Reply:
i was on the fence about including that detail as a feature, but thinking more about it, i think we should. I think it's useful to know when an instrumentation might impact a server span generated by another instrumentation. I'm going to followup and add a new `SERVER_SPAN_NAME` feature

Reply:
yea, i was just doing some digging and identified cxf, metro, and axis jaxws all fall into this category. I'm going to review more